launcher/src/main/java/org/apache/spark/launcher/package-info.java

```diff
@@ -49,6 +49,38 @@
  * </pre>
  *
  * <p>
+ * Currently, for currently while launching spark application with
+ * {@link org.apache.spark.launcher.SparkLauncher#startApplication(
+ * org.apache.spark.launcher.SparkAppHandle.Listener...)}, there are two options available
+ * for YARN manager in cluster deploy mode:
```

Comment:
we should either talk about the other option here or/and maybe in example above say it launches process not thread.